robottelo/ui/contentviews.py

```diff
@@ -42,7 +42,10 @@ def set_calendar_date_value(self, name, value):
         # the calendar panel popup and hide other form elements that became
         # unreachable.
         # close the popup calendar panel
-        self.click(locators.contentviews.calendar_date_button % name)
+        button = self.wait_until_element(
+            locators.contentviews.calendar_date_button % name, timeout=2)
+        if button:
```

Comment:
When sending data via selenium this popup is not always raised. But it works every time when checking manually, so not the problem on the Sat side. So I decided to add additional check before clicking rather than remove this part.